### PR TITLE
Update manpages from v3.3 to v3.10

### PR DESCRIPTION
**Pull request by presidento**
Using the man pages from [official admin guide](https://github.com/balabit/syslog-ng-ose-guides/).

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by presidento** [comment]
It is sad.

- The latest docbook stylesheet (`1.79.1`) generates broken manpages from our XMLs
- The internally used one (`1.78.1`) is not accessible on builder machines, so the build fails.

What should we do?

- Disable generating manpages during build?
- Generate broken pages?
- Postpone upgrading the very old man pages?

**Comment by fekete-robert** [comment]
Hi, I've managed to reproduce it, it seems to be a stylesheet issue, unless I'm missing something obvious. I'll try to investigate it in the next few days

**Comment by fekete-robert** [comment]
Using the stylesheets from the docbook-xsl-ns package (instead of docbook-xsl) seems to solve the problem

**Comment by presidento** [comment]
@fekete-robert thank you for the quick response, but unfortunately the result is the same as with `v1.78.1`: it does not compile.

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by mochrul** [comment]
I do recompile the man pages within the Debian package generation process.
And it still compiles (although with a lot of error/warning messages).

Of course, I only tried the old man pages.
```
sasa@sasa:~/src/debian/syslog-ng/git2/build-area/syslog-ng-3.10.1/debian/build-tree/doc/man$ ls -l
total 64
-rw-r--r-- 1 sasa sasa 11220 jún   29 08:01 loggen.1
-rw-r--r-- 1 sasa sasa 12934 jún   29 08:01 pdbtool.1
-rw-r--r-- 1 sasa sasa  8824 jún   29 08:01 syslog-ng.8
-rw-r--r-- 1 sasa sasa 15049 jún   29 08:01 syslog-ng.conf.5
-rw-r--r-- 1 sasa sasa  7254 jún   29 08:01 syslog-ng-ctl.1
```

**Comment by presidento** [comment]
@mochrul the mentioned bug only affects the updated manpages. With an ugly hack I managed to create good manpages - see my last commit -, so it is ready for review & merge.

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

File: tgz2build/rules [hunk 1]
```diff
@@ -82,7 +82,12 @@ $(STAMPDIR)/stamp-install: $(STAMPDIR)/stamp-build
 	$(INSTALL) -d -m 755 $(ZBS_STAGE_DIR)/$(ZBS_PREFIX)/share/man/man1
```
Comment (bazsi) [inline tgz2build/rules]: I don't think we are using tgz2build, do we?

Comment (presidento) [inline tgz2build/rules]: Me neither, but I am not sure of it. But the files were there so I had to update them.

**Pull request merged**